web/src/lib/components/faces-page/face-thumbnail.svelte

```diff
@@ -22,47 +25,57 @@
     border = false,
     onClick = () => {},
   }: Props = $props();
+
+  const handleAddToMerge = () => {
+    onClick(person);
+  };
 </script>
 
-<button
-  type="button"
-  class="relative rounded-lg transition-all"
-  onclick={() => onClick(person)}
-  disabled={!selectable}
-  style:width={thumbnailSize ? thumbnailSize + 'px' : '100%'}
-  style:height={thumbnailSize ? thumbnailSize + 'px' : '100%'}
->
+<div class="group relative" tabindex={selectable ? 0 : -1} role={selectable ? 'group' : undefined}>
   <div
-    class="h-full w-full border-2 brightness-90 filter"
-    class:rounded-full={circle}
-    class:rounded-lg={!circle}
-    class:border-transparent={!border}
-    class:dark:border-immich-dark-primary={border}
-    class:border-immich-primary={border}
+    class="relative rounded-lg transition-all"
+    style:width={thumbnailSize ? thumbnailSize + 'px' : '100%'}
+    style:height={thumbnailSize ? thumbnailSize + 'px' : '100%'}
   >
-    <ImageThumbnail {circle} url={getPeopleThumbnailUrl(person)} altText={person.name} widthStyle="100%" shadow />
-  </div>
-
-  <div
-    class="absolute start-0 top-0 h-full w-full bg-immich-primary/30 opacity-0"
-    class:hover:opacity-100={selectable}
-    class:rounded-full={circle}
-    class:rounded-lg={!circle}
-  ></div>
-
-  {#if selected}
     <div
-      class="absolute start-0 top-0 h-full w-full bg-blue-500/80"
+      class="h-full w-full border-2 brightness-90 filter"
       class:rounded-full={circle}
       class:rounded-lg={!circle}
-    ></div>
-  {/if}
+      class:border-transparent={!border}
+      class:dark:border-immich-dark-primary={border}
+      class:border-immich-primary={border}
+    >
+      <ImageThumbnail {circle} url={getPeopleThumbnailUrl(person)} altText={person.name} widthStyle="100%" shadow />
+    </div>
 
-  {#if person.name}
-    <span
-      class="w-100 text-white-shadow absolute bottom-2 start-0 w-full text-ellipsis px-1 text-center font-medium text-white hover:cursor-pointer"
+    {#if person.name}
+      <span
+        class="w-100 text-white-shadow absolute bottom-2 start-0 w-full text-ellipsis px-1 text-center font-medium text-white"
+      >
+        {person.name}
+      </span>
+    {/if}
+  </div>
+
+  {#if selectable}
+    <div
+      class="absolute left-1/2 -bottom-6 flex -translate-x-1/2 gap-2 opacity-0 transition-opacity group-hover:opacity-100 group-focus-within:opacity-100 [@media(hover:none)]:opacity-100"
```

Comment:
I think you could simplify this to `hidden hover:visible`, right?

Reply:
This would remove the fade in / fade out animation, but fine by me either way. Let me know what you prefer